litani

```diff
@@ -260,6 +260,11 @@ def get_args():
             "help":
                 "how many times to retry loading the run in 1 second intervals"
                 " (Default: %(default)s)"
+        }, {
+            "flags": ["--out-file"],
```

Comment:
This is a user-facing command, can I suggest we also have `"-o"` as a short flag? Please also update the documentation.

Reply:
Resolved